### PR TITLE
update eosti keymaps

### PR DESCRIPTION
**Pull request by eosti**
## Description

After getting a Planck, I needed to revamp my Iris and BDN9 layouts for ease of switching between. Now, both the Iris and the Planck have a very similar base layer, so I can quickly swap the two and not forget what all my layers do. Also adds licenses to all relevant files. 
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

*  N/A

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x]  I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/bdn9/keymaps/eosti/config.h [hunk 1]
```diff
@@ -1 +1,17 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
```
Comment (zvecr) [inline keyboards/keebio/bdn9/keymaps/eosti/config.h:16]:
```suggestion
#pragma once

```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/keebio/bdn9/keymaps/eosti/config.h (hunk 1 above)
Comment (eosti) [reply keyboards/keebio/bdn9/keymaps/eosti/config.h:16]: In a different PR, I was told that I should have a license in both the keymap.c and config.h files by drashana. Which is preferred?

Comment (zvecr) [reply keyboards/keebio/bdn9/keymaps/eosti/config.h:16]: Both the licence header, and the header guard

Comment (eosti) [reply keyboards/keebio/bdn9/keymaps/eosti/config.h:17]: Got it, thanks!

**Review by fauxpark — APPROVED** [review]

**Comment by eosti** [comment]
The Travis CI wasn't passing due to the compiled file size being too large, though it worked fine in my environment. I was a few versions behind though, so I suppose something updated and got larger. I just disabled a few things and now it comfortably fits.

File: keyboards/keebio/bdn9/keymaps/eosti/config.h [hunk 2]
```diff
@@ -1 +1,20 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#define TAPPING_TERM 275
+#define NO_DEBUG
+#define NO_PRINT
```
Comment (zvecr) [inline keyboards/keebio/bdn9/keymaps/eosti/config.h:20]: If trying to save space, its nicer to add a keymap level `rules.mk` with 
```c
CONSOLE_ENABLE = no
COMMAND_ENABLE = no
```

```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by zvecr — APPROVED** [review]

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**